src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -188,15 +189,28 @@ const keriNotificationsChangeHandler = async (
   event: KeriNotification,
   dispatch: ReturnType<typeof useAppDispatch>
 ) => {
-  dispatch(
-    setQueueConnectionCredentialRequest({
-      id: event?.id,
-      type: ConnectionCredentialRequestType.CREDENTIAL_OFFER_RECEIVED,
-      logo: "", // TODO: must define Keri logo
-      label: "Credential Issuance Server", // TODO: must define it
-      source: ConnectionType.KERI,
-    })
-  );
+  if (!event) {
+    return;
+  }
+  if (event.a.r === NotificationRoute.Credential) {
+    dispatch(
+      setQueueConnectionCredentialRequest({
+        id: event?.id,
+        type: ConnectionCredentialRequestType.CREDENTIAL_OFFER_RECEIVED,
+        logo: "", // TODO: must define Keri logo
+        label: "Credential Issuance Server", // TODO: must define it
+        source: ConnectionType.KERI,
+      })
+    );
+  } else if (event.a.r === NotificationRoute.MultiSigIcp) {
+    //TODO: Use dispatch here, handle logic for the multisig notification
+    // const meta = {
+    //   theme: 4,
+    //   colors: ["#000000", "#000000"],
+    //   displayName: `Multisig-${new Date().getTime()}`,
+    // } as any;
+    // await AriesAgent.agent.identifiers.joinMultisig(event.id as string, meta)
```

Comment:
commented out code

Reply:
@iFergal I think we should pop up a screen for users to accept the join request first. Please let me know what you think about this.

Reply:
Yeah, we should. I was going to say we should just auto-join for now and then create a pop up in another story, but if we're open sourced maybe we should avoid that.

Can we still remove the commented out code to keep it clean, since the actual join will be in a pop-up?

Just leave the todo :)

Reply:
Yes. I will remove the comment.